microcosm_resourcesync/schemas/hal_schema.py

```diff
@@ -5,12 +5,31 @@
 from microcosm_resourcesync.following import FollowMode
 from microcosm_resourcesync.schemas.base import Link, Schema
 
+HTTP_EXCLUDED_KEYS = ["id", "_links"]
+
 
 class HALSchema(Schema):
     """
     A schema that implements HAL JSON linking.
 
     """
+    def to_http_data(self):
```

Comment:
I might say something like "to_replace_operation_schema" (probably not that exact name), but I feel like `http` is misleading here. The comment below is super helpful, the name just bothers me.

It remains to be seen how much we use resource sync in the future, but if we were to end up using this more heavily, for each resource we are trying to sync, I might look up the swagger schema for that resource from the service we are syncing it to, and use that schema to explicitly use the `dump` function to make sure the right fields get included.

I'm just thinking that in general lots of our resources have computed fields which we won't be able to capture with a static list of excluded keys.

Note: I am not recommending you try to do this now. It doesn't look like there's any reason to put that much effort in, just mentioning for posterity and trying to come up with a better name for now.